parsl/tests/test_checkpointing/test_python_checkpoint_3.py

```diff
@@ -8,6 +8,7 @@
 
 
 def local_setup():
+    parsl.clear()
```

Comment:
if this is needed, another test hasn't cleared up after itself properly, I think

Reply:
I've removed this.